MPSGameLift/Code/Source/MPSGameLiftModuleInterface.h

```diff
@@ -20,6 +22,7 @@
     #include <AzCore/Console/IConsole.h>
 #endif
 
+#pragma optimize("",off)
```

Comment:
I understand this PR is WIP, but feels a tad risky to commit this at all (same for line 88)

Reply:
Ah, good catch thank you!